src/v2/Roboto_Regular.ufo/features.fea

```diff
@@ -939,7 +939,26 @@ feature liga {
     sub f l by f_l;
   } liga02;
 
-  language TUR  exclude_dflt;
+  # include all ligatures for these languages
+  language FRA;  # French
+  lookup liga01;
+  lookup liga02;
+  language MOL;  # Moldavian
+  lookup liga01;
+  lookup liga02;
+  language NAV;  # Navajo
+  lookup liga01;
+  lookup liga02;
+  language ROM;  # Romanian
+  lookup liga01;
+  lookup liga02;
```

Comment:
I think they are; the problem with that snippet for example is that the fi ligature is included in scripts other than latn.